sequencing-server/src/app.ts

```diff
@@ -64,11 +64,13 @@ app.use(async (_: Request, res: Response, next: NextFunction) => {
 
   const activitySchemaDataLoader = new DataLoader(activitySchemaBatchLoader({ graphqlClient }), {
     cacheKeyFn: objectCacheKeyFunction,
+    name: null,
```

Comment:
can you tell me what the new name fields are for

Reply:
`The name given to this `DataLoader` instance. Useful for APM tools.` Just pulled from the docs, I don't think we need it but it's required as an option.